src/core/batch/parquetSchema.ts

```diff
@@ -80,6 +109,55 @@ export const ReactionSchema = {
   signature: { type: "BYTE_ARRAY" },
 };
 
+/**
+ * ReactionBloomFilter: bloom filter options for batching reaction messages
+ */
 export const ReactionBloomFilterOptions = {
   bloomFilters: [{ column: "emoji" }, { column: "fromId" }, { column: "inReplyTo" }],
 };
+
+/**
+ * getSchemaFor() takes DSNPType and returns its corresponding parquet schema
+ *
+ * @param   dsnpType a dsnpType
+ * @returns The corresponding parquet schema
+ */
+export const getSchemaFor = (dsnpType: DSNPType): Schema => {
+  switch (dsnpType) {
+    case DSNPType.GraphChange:
+      return GraphChangeSchema;
+    case DSNPType.Broadcast:
+      return BroadcastSchema;
+    case DSNPType.Reply:
+      return ReplySchema;
+    case DSNPType.Reaction:
+      return ReactionSchema;
+    case DSNPType.Profile:
+      return ProfileSchema;
+    default:
+      throw new Error(`Invalid DSNP type: ${dsnpType}`);
+  }
+};
+
+/**
+ * getBloomFilterOptionsFor() takes DSNPType and returns its bloom filter options
+ *
+ * @param   dsnpType a dsnpType
+ * @returns The corresponding parquet bloom filter options
+ */
+export const getBloomFilterOptionsFor = (dsnpType: DSNPType): BloomFilterOptions => {
+  switch (dsnpType) {
+    case DSNPType.GraphChange:
+      return GraphChangeBloomFilterOptions;
+    case DSNPType.Broadcast:
+      return BroadcastBloomFilterOptions;
+    case DSNPType.Reply:
+      return ReplyBloomFilterOptions;
+    case DSNPType.Reaction:
+      return ReactionBloomFilterOptions;
+    case DSNPType.Profile:
+      return ProfileBloomFilterOptions;
+    default:
+      throw new Error(`Invalid DSNP type: ${dsnpType}`);
+  }
```

Comment:
```suggestion
  }
  throw new Error(`Invalid DSNP type: ${dsnpType}`);
```

Reply:
It is in TypeScript, but in just ol' JS it will not be if someone is using this direct as they can throw any number at it they want to... Although I guess that would just return an undefined. Also, now that I think about it: Is this accessible outside of the sdk?